checks/autologin.go

```diff
@@ -1,10 +1,11 @@
 package checks
 
 import (
-	"os"
 	"os/exec"
 	"path/filepath"
 	"strings"
+
+	"paretosecurity.com/auditor/shared"
```

Comment:
Can we stop using the "auditor" term? We no longer have the "updated" so having "auditor" does not make sense.

Reply:
checker? It needs to be something.

Reply:
linux?

Reply:
Actually we now have public repo so module name is repo name.